docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -59,15 +60,30 @@ The important things to keep in mind are:
  
 If you don’t see information that could be helpful on this guide, please let us know through [our Jira Service Manager](https://citz-do.atlassian.net/servicedesk/customer/portal/2).
 
-## Technical information 
+## Technical Transition Guide
+
+Our technical transition guide helps GitHub users prepare for the upgrade, and confirms if users completed linking successfully. [Our IDIR transition guide](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/) offers more details about IDIRs. 
+
+After **July 30th, 2024**, users cannot access the bcgov GitHub organization if they didn’t link their IDIR and GitHub ID. There are mechanisms in the system that force an IDIR login when you use your GitHub to login. 
 
-Our technical transition guide offers more details and helps GitHub users prepare for the upgrade.
+### Validating authentication
 
-The video shows the process, starting from the banner link on the bcgov GitHub organization, to the end:
+GitHub doesn't offer users a screen to confirm authentication. There are 3 alternative ways to validate linking GitHub IDs and IDIRS: 
+
+1) The video below shows the process from start to finish. 
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/-cfhUY_or8s?si=q0pjaxYd4FBhNZ0s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 
-After **July 30th, 2024**, users cannot access the bcgov GitHub organization if they didn’t link IDIRs and GitHub IDs. During here are mechanisms in the system that force an IDIR login when you use your GitHub to login. 
+2) If users **don't** have the "**Authenticate your account**" button on their dashboards, then authentication is complete.
```

Comment:
Ah shoot forgot to circle back with you on this! I misheard shea originally - this is a dismissible banner for some silly reason, so there's a chance that if someone closed the notification banner out of instinct they may miss this.

Reply:
Okay, should I remove this completely? Deleting it would impact the other comments.